static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+
+import {AreaChart, AreaChartProps} from 'sentry/components/charts/areaChart';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {formatBytesBase2} from 'sentry/utils';
+import theme from 'sentry/utils/theme';
+
+function MemoryChart({memorySpans}) {
+  const chartOptions: Omit<AreaChartProps, 'series'> = {
+    grid: {
+      left: '10px',
+      right: '10px',
+      top: '40px',
+      bottom: '10px',
+    },
+    tooltip: {
+      trigger: 'axis',
+      valueFormatter: (value: number | null) => formatBytesBase2(value || 0),
+    },
+    xAxis: {
+      type: 'time',
+      min: memorySpans[0]?.data?.timestamp,
+      max: memorySpans.slice(-1)?.data?.timestamp,
+    },
+    yAxis: {
+      type: 'value',
+      name: t('Heap Size'),
+      nameTextStyle: {
+        padding: 8,
+        fontSize: theme.fontSizeLarge,
+        fontWeight: 600,
+        lineHeight: 1.2,
+        color: theme.gray100,
+      },
+      min: 0,
+      max: memorySpans.slice(-1)?.data?.memory.totalJSHeapSize,
+      axisLabel: {
+        formatter: value => formatBytesBase2(value),
+      },
+    },
+  };
+
+  const series = [
+    {
+      seriesName: t('Used Heap Memory'),
+      data: memorySpans.map((span, i) => ({
+        value: span.data.memory.usedJSHeapSize,
+        name: i,
```

Comment:
Hmm its been awhile since I've used the charts, we could do some pairing tomorrow though?

When you say they aren't playing right, what do you mean? Is it an issue with the units? e.g. seconds vs ms?

Reply:
Oh! I bet it's because the timestamps are huge numbers, so they probably are stuck like way off the chart if I had to guess.

Reply:
Nope I was wrong, I was accessing it via span.data.timestamp when it should have been span.timestamp. Pushing a fix once I format the times to something that isn't a bagillion seconds.